terraform/modules/happy-stack-eks/main.tf

```diff
@@ -5,36 +5,55 @@ data "kubernetes_secret" "integration_secret" {
   }
 }
 
+resource "validation_error" "mix_of_internal_and_external_services" {
```

Comment:
Now that we are on updated versions of Terraform, we can use this: https://developer.hashicorp.com/terraform/language/expressions/custom-conditions instead of a custom terraform provider to validate our input variables. Can we update these validations to use input variable validation instead?